### PR TITLE
fix to library of device Id

### PR DESCRIPTION
**Pull request by BRAYANP**

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -18,13 +18,16 @@ var userToken = require('./util').token;
 module.exports = {
   startDeviceRegister: function (options, events) {
     //Just using userToken was returning an error
+    //console.log('BRAYAN >>>> helpers.js', options);
```
Comment (diegoribero) [inline lib/helpers.js:21]: Not necessary

File: lib/queue.js [hunk 2]
```diff
@@ -10,32 +10,25 @@ var refs = require('./util').refs;
...
   application_create: {
-    error_state: 'error',
-    in_progress_state: 'application-create-in-progress',
+    error_state: "error",
+    in_progress_state: "application-create-in-progress",
     retries: 3,
-    start_state: 'application-create',
-    timeout: 15000
-  },
-  application_uninstall: {
```
Comment (diegoribero) [inline lib/queue.js:34]: No need to delete application_uninstall

File: lib/queue.js [hunk 3]
```diff
@@ -60,23 +53,36 @@ var processTask = function (spec, options, events) {
...
-
-    debug('Task Queue>', task)
-
-    if (_.isNull(task) || _.isUndefined(task) || (spec.hasOwnProperty('finished_state') && task._state == spec.finished_state)) {
+    if (_.isNull(task) || _.isUndefined(task) ) {
       finished();
+    }
+    else if ((spec.hasOwnProperty('finished_state') && task._state == spec.finished_state)) {
+      var paramsToSend;
+      //TODO check if finished exists
```
Comment (diegoribero) [inline lib/queue.js:75]: Already checked

**Comment by eighteyes** [comment]
@BRAYANP do you want this to still get merged? it has not been updated. It has conflicts

**Pull request closed without merging**